crates/provider-sqldb-postgres/wit/connection.wit

```diff
@@ -0,0 +1,61 @@
+package wasmcloud:postgres@0.1.0-draft;
+
+/// Manual connection management
```

Comment:
See my note in the README for my thoughts on manual connection

Reply:
Saw this! Will resolve in favor of the other comment for discussion!